server/service/handler.go

```diff
@@ -799,10 +799,6 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 		errorLimiter.Limit("get_device_mdm", desktopQuota),
 	).GET("/api/_version_/fleet/device/{token}/mdm/apple/manual_enrollment_profile", getDeviceMDMManualEnrollProfileEndpoint, getDeviceMDMManualEnrollProfileRequest{})
 
-	demdm.WithCustomMiddleware(
```

Comment:
Is it possible that older versions of `fleetd` would still try to hit this endpoint?

Reply:
yes! good eye, but according to this guide is a nice to have, not a must https://github.com/fleetdm/fleet/blob/13157-fv-escrow/docs/Contributing/fleetd-development-and-release-strategy.md